src/screens/NewContactAddScreen/index.tsx

```diff
@@ -70,9 +57,31 @@ const NewContactAddScreen: FC<Props> = (props: Props): ReactElement => {
     }, [onBackPress]),
   );
 
+  useEffect((): void => {
+    if (!federations) {
+      return;
+    }
+    const branded = federations.map(federation => fetchBrandingForContact(federation));
+    Promise.all(branded).then(result => setBrandedFederations(result));
+  }, []);
```

Comment:
Is federations updatable?
-->
```
}, [federations]);

Reply:
no it is not, i need to look at this later anyways for the issue where it is untrusted before checking anyways. so will make changes then